src/vs/workbench/contrib/webview/electron-browser/webviewElement.ts

```diff
@@ -315,6 +315,7 @@ export class WebviewElement extends Disposable implements Webview {
 
 		this._webview = document.createElement('webview');
 		this._webview.setAttribute('partition', `webview${Date.now()}`);
+		this._webview.setAttribute('nodeintegration', 'yes');
```

Comment:
We don't want to enable node in webviews. Shouldn't the `preload` script still have node integration enabled while it is disabled for the webview content itself?

Reply:
Good catch! yeah this is not needed, the preload script will have access to node environment.